src/Impostor.Server/Net/State/Game.Data.cs

```diff
@@ -239,7 +241,17 @@ public async ValueTask<bool> HandleGameDataAsync(IMessageReader parent, ClientPl
                             return false;
                         }
 
-                        sender.Scene = reader.ReadString();
+                        // According to game assembly, sender is only allowed to send OnlineGame.
```

Comment:
Don't completely agree, if I look at AmongUsClient.CoOnPlayerChangedScene it seems like it has two values where it does something useful: OnlineGame and Tutorial.

Reply:
What about mods? Do they ever add / change scenes?

Reply:
I'm not aware of any. Even for custom maps it's not necessary.

Reply:
It will only send Tutorial if the player somehow manages to connect to Tutorial while being in online

Reply:
Idk tbh, as far as i've tested not even in this way, only by hacking